src/components/HomepageUpsell/HomepageUpsell.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Fragment } from 'react';
+import { StyledButton, BodyText, MapWrapper } from './HomepageUpsell.style';
+import { EventAction, EventCategory } from 'components/Analytics';
+import Map from 'components/Map/Map';
+import LabelWithChevron from 'components/NewLocationPage/Shared/LabelWithChevron';
+import {
+  Experiment,
+  ExperimentID,
+  Variant,
+  VariantID,
+} from 'components/Experiment';
+
+const HomepageUpsell = () => {
+  return (
+    // Swap A and B before merging:
+    <Experiment id={ExperimentID.HOMEPAGE_UPSELL}>
+      <Variant id={VariantID.B}>
+        <Fragment />
+      </Variant>
+      <Variant id={VariantID.A}>
+        <StyledButton
+          trackingCategory={EventCategory.EXPERIMENT}
+          trackingAction={EventAction.CLICK}
```

Comment:
Maybe `EventAction.NAVIGATION` would be better here? I'm not sure if we want to have the `EXPERIMENT` category, because if this gets promoted to production we will need to change the category